net.sourceforge.Pdfedit.json

```diff
@@ -0,0 +1,173 @@
+                        "fedora_rpm_patches/qt-x11-free-3.3.8b-uic-multilib.patch",
+                        "fedora_rpm_patches/qt-3.0.5-nodebug.patch",
+                        "fedora_rpm_patches/qt-3.1.0-makefile.patch",
+                        "fedora_rpm_patches/qt-3.3.3-gtkstyle.patch",
+                        "fedora_rpm_patches/qt-3.3.4-print-CJK.patch",
+                        "fedora_rpm_patches/qt-3.3.6-fontrendering-214371.patch",
+                        "fedora_rpm_patches/qt-3.3.6-fontrendering-gu-228452.patch",
+                        "fedora_rpm_patches/qt-3.3.6-fontrendering-ml_IN-209097.patch",
+                        "fedora_rpm_patches/qt-3.3.6-fontrendering-ml_IN-209974.patch",
+                        "fedora_rpm_patches/qt-3.3.6-fontrendering-ml_IN-217657.patch",
+                        "fedora_rpm_patches/qt-3.3.6-fontrendering-te_IN-211259.patch",
+                        "fedora_rpm_patches/qt-3.3.8-fontrendering-#214570.patch",
+                        "fedora_rpm_patches/qt-3.3.8-fontrendering-as_IN-209972.patch",
+                        "fedora_rpm_patches/qt-uic-nostdlib.patch",
+                        "fedora_rpm_patches/qt-x11-free-3.3.2-quiet.patch",
+                        "fedora_rpm_patches/qt-x11-free-3.3.3-gl.patch",
+                        "fedora_rpm_patches/qt-x11-free-3.3.3-qembed.patch",
+                        "fedora_rpm_patches/qt-x11-free-3.3.4-fullscreen.patch",
+                        "fedora_rpm_patches/qt-x11-free-3.3.5-uic.patch",
+                        "fedora_rpm_patches/qt-x11-free-3.3.6-qfontdatabase_x11.patch",
+                        "fedora_rpm_patches/qt-x11-free-3.3.6-strip.patch",
+                        "fedora_rpm_patches/qt-x11-free-3.3.7-arm.patch",
+                        "fedora_rpm_patches/qt-x11-free-3.3.7-umask.patch",
+                        "fedora_rpm_patches/qt-x11-free-3.3.8-odbc.patch"
+                    ]
+                },
+                {
+                    "type": "patch",
+                    "strip-components": 0,
+                    "options": ["-l"],
+                    "paths": [
+                        "fedora_rpm_patches/0079-compositing-types.patch",
+                        "fedora_rpm_patches/0080-net-wm-sync-request-2.patch",
+                        "fedora_rpm_patches/0084-compositing-properties.patch",
+                        "fedora_rpm_patches/qt-3.3.8-libpng15.patch"
+                    ]
+                },
+                {
+                    "type": "patch",
+                    "path": "qt3_flatpak_build_fixes.patch"
+                }
+            ]
+        },
+        {
+            "name": "boost",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./bootstrap.sh",
+                "./b2 install --prefix=/app --with-system --with-random define=\"BOOST_SYSTEM_NO_DEPRECATED\" -sNO_BZIP2=1 cxxflags=-fPIC cflags=-fPIC"
```

Comment:
Should be limited to libraries the app actually uses.

Reply:
Yes, PDFedit uses this library.